packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -175,7 +224,9 @@ const Statement = ({
                             empty_message_component={EmptyTradeHistoryMessage}
                             component_icon={component_icon}
                             localized_message={localize('You have no transactions yet.')}
-                            localized_period_message={localize('You have no transactions for this period.')}
+                            localized_period_message={localize(
+                                "You've made no transactions of this type during this period."
```

Comment:
Please use single quotes

Reply:
I tried to use singlequotes 
like this 

`You\'ve made no transactions of this type during this period.'`

but somehow when i run git commit, it will be override by the auto-styling because of the backslash

Reply:
Yep, ignore my comment. :)